packages/cozy-pouch-link/package.json

```diff
@@ -13,8 +13,10 @@
   "dependencies": {
     "cozy-client": "^13.19.0",
     "cozy-device-helper": "^1.7.3",
+    "debug": "^4.1.1",
```

Comment:
Is is really intended ?

Reply:
It was because it didn't work without. But I just checked the package.json https://unpkg.com/pouchdb-debug@7.2.1/package.json and it seems to be a dep. Let me retry without it